index.html

```diff
@@ -30,7 +31,77 @@
       <p id="connect">LET’S CONNECT</p>
       <img id="icons" src="assets/images/social-media-icons.svg" alt="social-media icons">
   </section>
-    
-    
+</div>
+   
+
+  <div id="section1" class="section_2">
+    <img id="snap" src="assets/images/Snapshoot_Portfolio.svg" alt="Snapshot of portfolio">
+    <img id="tonic" src="assets/images/tonic.png" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </div>
+
+  <section id="section2" class="section_2">
+    <img class="snap2" src="assets/images/Snap2.svg" alt="Snapshot of portfolio">
+    <img class="stories" src="assets/images/stories.svg" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </section>
+
+  <section id="section3" class="section_2">
+    <img class="snap2" src="assets/images/tonic2.svg" alt="Snapshot of portfolio">
+    <img id="second_tonic" src="assets/images/tonic.png" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </section>
+
+  <section id="section4" class="section_2">
+    <img class="snap2" src="assets/images/stories2.svg" alt="Snapshot of portfolio">
+    <img class="stories" src="assets/images/stories.svg" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </section>
```

Comment:
- [ ] for semantics  and SEO reasons, kindly wrap all the cards inside a `<section>` wich has a display:grid, and replace each card with `<div>` instead of sections

Reply:
I have wrapped cards in <section>  and each card in a <div>.

Reply:
@Bludidi each card should be a `<div>` and all cards should be in one section